tests/run_tests.py

```diff
@@ -188,23 +205,38 @@ def main():
 
     # system
     if any(i in ['all', 'system'] for i in args.tests):
-        test_results['system'] = run_system(args.system, vic_exe, data_dir,
-                                            os.path.join(out_dir, 'system'),
-                                            args.driver)
+        if len(args.driver) == 1:  # if only one driver
+            driver = args.driver[0]
+            test_results['system'] = run_system(args.system, vic_exe, data_dir,
+                                                os.path.join(out_dir,
+                                                             'system'),
+                                                driver)
+        else:  # if multiple drivers
+            list_driver = args.driver
+            test_results['system'] = run_system(args.system, list_vic_exe,
+                                                data_dir,
+                                                os.path.join(out_dir,
+                                                             'system'),
+                                                list_driver)
     # science
     if any(i in ['all', 'science'] for i in args.tests):
-        test_results['science'] = run_science(args.science, vic_exe,
-                                              science_test_data_dir,
-                                              data_dir,
-                                              os.path.join(out_dir, 'science'),
-                                              args.driver)
+        if len(args.driver) == 1:  # if only one driver
+            driver = args.driver[0]
+            test_results['science'] = run_science(
+                                            args.science, vic_exe,
+                                            science_test_data_dir,
+                                            data_dir,
+                                            os.path.join(out_dir, 'science'),
+                                            driver)
```

Comment:
I would make sure you are only running with the classic driver here. Here is a place where the dictionary approach I discussed above would help.